### ISSUE TITLE
Disable auto-hyperlinking in Gmail editable windows

### ISSUE THREAD

**Issue by NickMOrlando** [TheSoftwareDev/JiraSearchTool#5]
When typing into an editable text field (e.g. replying to an email in Gmail/Inbox), the auto-hyperlinking is triggering during edits (and the local chrome-extension:// hyperlink ends up being included in the outbound message)

**Comment by erwa** [comment]
I think this auto-linking a cool idea, but I would have expected it to link to the actual JIRA ticket (e.g.: https://issues.apache.org/jira/browse/HADOOP-3629) rather than to the chrome-extension page. Also, the auto-hyperlinking happens too quickly, as soon as I type HADOOP-3 rather than the full JIRA id.

**Comment by NickMOrlando** [comment]
Are you seeing the link to the chrome-extension page after you save the URL on the options page?

**Comment by erwa** [comment]
Yeah, even after I've changed the URL on the options page, the link is still chrome-extension://jlpbojfbbmodkbjhemkcmkhhlkpijhbi/background.html?jira=XXX-123.

Also, after the link is created, why does the cursor move to the beginning of the link? It should remain at the end of the link so I continue typing my sentence.
